maps/graph_edges/glitched_logic/iwa_parakarryless_mt_rugged_seed_clippy.py

```diff
@@ -0,0 +1,8 @@
+"""
+This file represents edges of the world graph that have to be added
+for Glitched Logic: Parakarry-less Mt. Rugged Seed using Clippy.
+"""
+edges_iwa_add_parakarryless_mt_rugged_seed_clippy = [
+    #* Mt Rugged 3 Exit Bottom Left -> GiftA (MagicalSeed2)
+    {"from": {"map": "IWA_02", "id": 0},"to": {"map": "IWA_02", "id": "GiftA"}, "reqs": [["Boots"], ["Lakilester"]], "mapchange" : False},
+]
```

Comment:
Would it make sense to also add an edge here `IWA_02/2 > IWA_02/GiftA` with reqs=`[["Lakilester"]]`?
By coming from the desert side and getting clippy off of the cleft at the top, you could squeeze through the "Early Seed" gap in the geometry, like happened to you here https://www.twitch.tv/phantom5800/clip/SmellyRepleteJellyfishPicoMause-HWVoq9EnVzDwd_ny

Reply:
I'm not sure how precise this is, but I don't see a reason not to add this